spark/src/test/java/org/apache/zeppelin/spark/SparkInterpreterTest.java

```diff
@@ -144,10 +145,16 @@ public void testZContextDependencyLoading() {
 
   @Test
   public void emptyConfigurationVariablesOnlyForNonSparkProperties() {
-    for (Tuple2<String, String> tuple2 : repl.getSparkContext().getConf().getAll()) {
-      if (tuple2._1().startsWith("spark."))
-        assertFalse(String.format("configuration starting from 'spark.' should not be empty. [%s]: [%s]", tuple2._1(), tuple2._2()), tuple2._2().isEmpty());
+    Properties intpProperty = repl.getProperty();
+    SparkConf sparkConf = repl.getSparkContext().getConf();
+    for (Object oKey : intpProperty.keySet()) {
+      String key = (String) oKey;
+      String value = (String)intpProperty.get(key);
+      repl.logger.debug(String.format("[%s]: [%s]", key, value));
+      if (key.startsWith("spark.") && value.isEmpty()) {
+        if (value.isEmpty())
```

Comment:
I'm confused, on what condition this can be false and execution even reaches this place?